conversion_sr14_sr15/README.md

```diff
@@ -0,0 +1,78 @@
+# 1.0 Introduction
+The _conversion_sr14_sr15_ script converts Series 14 (SR14) data, used primarily in ABM2+, into Series 15 (SR15) format, which is used in ABM3. It adjusts SR14 land use and synthetic population files to match SR15 MGRA boundaries and uses an existing SR15 land use data to estimate SR15-equivalent land use columns. The exact processing procedures implemented are detailed in the following section. 
+
+# 2.0 Methodology
+## 2.1 Input Files
+Table 1 lists the input files required to run the conversion script and their purpose in the process. All files must be for the same scenario year. For clarity, SR15 refers to the converted SR14 file (output of this script), while ABM3 refers to the existing ABM3 land use input file (in SR15 format) that is used in the conversion process.
+
+**Table 1. Input Files**
+|File Name|Purpose|
+|---|---|
+|SR14 synthetic household file|Recoded to SR15 MGRAs and fields renamed consistent with ABM3 field names. Also summarized for household totals to be used in converted MGRA data. |
+|SR14 synthetic person file| Fields renamed consistent with ABM3 field names. Summarized for person totals to be used in converted MGRA data.| 
+|SR14 land use file| Source of employment and income to be used in converted MGRA data file.|
+|ABM3 land use| Source of SR15 MGRAs, TAZs, acres, park space, parking costs, and a few other fields held constant in converted MGRA data.|
+|MGRA SR14 to SR15 crosswalk| Crosswalk used for conversion. Created via a separate process in which the centroid of each SR14 MGRA was geocoded to the polygon of SR15 MGRAs.|
+|TAZ SR14 to SR15 crosswalk| Crosswalk used for conversion.|
+|SR15 synthetic person file| Generates naics2_original_code and soc2 columns for SR15 synthetic person file by drawing a uniform random sample.|
+
+## 2.2 Data Processing
+### Households and Persons 
+The synthetic household and person files undergo minimal processing. For the household file, the MGRA and TAZ columns are updated using the SR14 to SR15 crosswalks, and the column “hworkers” is renamed to “num_workers”. For the person file, "naics2_original_code" and "soc2" are generated by drawing a uniform random sample from an existing SR15 synthetic person file. The converted SR15 person and household files are used to calculate household and person totals in the converted SR15 land use file. 
```

Comment:
For the method of generating 'naics2_original_code' and 'soc2', does the uniform random sample change every time while the tool is running? I wonder if the outputs are different each time with the same input files?

Reply:
And just quick check the output 'persons' file, and I found the  'naics2_original_code' and 'soc2' have 0 values with workers (employ = [1,2]). I don't think it's correct. Could you check your code and make an explanation? @ednaaguilar